files/sidebars/gamessidebar.yaml

```
@@ -195,18 +203,19 @@ l10n:
     Audio_for_Web_Games: Audio for Web Games
     2D_collision_detection: 2D collision detection
     Tiles_and_tilemaps_overview: Tiles and tilemaps overview
+    Controls_Gamepad_API: Implementing controls using the Gamepad API
+    Crisp_pixel_art_look: Crisp pixel art look with image-rendering
     3D_games_on_the_Web: 3D games on the Web
-    3D_games_on_the_Web_overview: 3D games on the Web overview
     Explaining_basic_3D_theory: Explaining basic 3D theory
     Building_up_a_basic_demo_with_A-Frame: Building up a basic demo with A-Frame
     Building_up_a_basic_demo_with_Babylon.js: Building up a basic demo with Babylon.js
     Building_up_a_basic_demo_with_PlayCanvas: Building up a basic demo with PlayCanvas
     Building_up_a_basic_demo_with_Three.js: Building up a basic demo with Three.js
+    GLSL_Shaders: GLSL Shaders
```

```suggestion
    GLSL_Shaders: GLSL shaders
```
Need to be propagated to other locales